setup.py

```diff
@@ -32,7 +32,7 @@ def read_file(filename, encoding='utf8'):
 
 install_requires = [
     'acme=={0}'.format(version),
-    'ConfigArgParse',
+    'ConfigArgParse>=0.10.0',  # python2.6 support, upstream #17
```

Comment:
Well, if that causes the problem, then please open a separate bug to discuss changing our strategy. But this PR is not the best place to discuss is, or deviate from already established standards.

One potential harm that instantly comes to my mind is that it makes OS packaging a bit harder (maintainers, such as @hlieberman, will have to patch `setup.py` unnecessarily).